projects/baby_driver/scripts/gpio_port.py

```diff
@@ -0,0 +1,15 @@
+"""Constants for identifying GPIO ports."""
+
+import enum
+
+class GpioPort(enum.Enum):
```

Comment:
Ah, it seems like some of the tests are failing because inheriting from `Enum` makes e.g. `GpioPort.A` not equal to 0, instead it's some weird enum object. I think you should just leave this as a normal class and not have it inherit from Enum.